src/hotspot/share/runtime/monitorDeflationThread.hpp

```diff
@@ -41,6 +41,8 @@ class MonitorDeflationThread : public JavaThread {
 
   // Hide this thread from external view.
   bool is_hidden_from_external_view() const { return true; }
+
+  bool is_monitor_deflation_thread() const override  { return true; }
```

Comment:
Don't say "override" here, because clang would then trigger lots of warnings for methods that were not marked as such. Yes, it is unfortunate, but...